docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -0,0 +1,7 @@
+#Internal and external communication
+
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For example, as mentioned in [[Polyglot systems with Lagom|PolyglotSystems]], legacy application components might use synchronous communication that cannot be changed.  When choosing the mode of communication for your microservices, keep in mind inter-service communication as well as external communication. 
```

Comment:
The distinction between "inter-service" and "external" isn't totally clear. Should it be "intra-service"? Or does "external" mean "with the internet"?

I guess this page is meant to capture some of my big blue comment on the draft page, but I think that it might need to be expanded, and maybe we should leave it out of this revision until more content is written.